SimTKcommon/Geometry/src/DecorativeGeometryRep.h

```diff
@@ -568,12 +568,20 @@ class DecorativeMeshFileRep : public DecorativeGeometryRep {
 // no default constructor
 explicit DecorativeMeshFileRep(const std::string& meshFileName) 
 :   meshFile(meshFileName) {
+    pmesh = nullptr;
 }
 
 const std::string& getMeshFile() const {
     return  meshFile;
 }
 
+const PolygonalMesh& getMesh() {
```

Comment:
This method will throw an error if the file extension is not recognized. That could be a surprise to someone who doesn't understand that the file is lazy-loaded. This behavior should be documented at the top level since it will be user-visible. Please consider whether it would make sense to validate the file extension when it is first seen rather than waiting until lazy loading -- that could avoid some surprises. But that wouldn't get rid of all possible errors since the file might not be present or readable.